lib/features/email/domain/usecases/move_to_mailbox_interactor.dart

```diff
@@ -28,14 +28,14 @@ class MoveToMailboxInteractor {
       final result = await _emailRepository.moveToMailbox(session, accountId, moveRequest);
       if (result.isNotEmpty) {
         yield Right(MoveToMailboxSuccess(
-          result.first,
-          moveRequest.currentMailboxes.keys.first,
-          moveRequest.destinationMailboxId,
-          moveRequest.moveAction,
-          moveRequest.emailActionType,
-          destinationPath: moveRequest.destinationPath,
-          currentMailboxState: currentMailboxState,
-          currentEmailState: currentEmailState));
+            result.first,
+            moveRequest.currentMailboxes.keys.first,
+            moveRequest.destinationMailboxId,
+            moveRequest.moveAction,
+            moveRequest.emailActionType,
+            destinationPath: moveRequest.destinationPath,
+            currentMailboxState: currentMailboxState,
+            currentEmailState: currentEmailState));
```

Comment:
Please don't format it.

Reply:
I was just removing old functionality, but i will fix it.